apps/dashboard/src/modules/attendance/components/PoolForm/CreatePoolForm.tsx

```diff
@@ -0,0 +1,68 @@
+import { type AttendancePool } from "@dotkomonline/types"
+import { Box, Button } from "@mantine/core"
+import { type FC } from "react"
+import { z } from "zod"
+import { validateAndReturn } from "./utils"
+import { createLabelledCheckboxGroupInput, createNumberInput, useFormBuilder } from "../../../../app/form"
+import { notifyFail } from "../../../../app/notifications"
+
+export interface CreatePoolFormProps {
+  onSubmit(values: CreatePoolFormSchema): void
+  attendancePools: AttendancePool[]
+  onClose(): void
+  defaultValues: CreatePoolFormSchema
+  mode: "create" | "update"
+}
+
+export const CreatePoolFormSchema = z.object({
+  yearCriteria: z.array(z.number()),
+  limit: z.number(),
+})
+export type CreatePoolFormSchema = z.infer<typeof CreatePoolFormSchema>
+
+export const useCreatePoolFormLogic = (props: CreatePoolFormProps) => {
```

Comment:
I think adding "Logic" here adds little meaning, I suggest we shorten :D

```suggestion
export const useCreatePoolForm = (props: CreatePoolFormProps) => {
```